tensorflow-horovod-dist-train.md

```diff
@@ -0,0 +1,112 @@
+# Distributed training using TensorFlow and Horovod on Amazon EKS 
+
+This document explains how to perform distributed training on [Amazon EKS](https://aws.amazon.com/eks/) using TensorFlow and Horovod with synthetic dataset. 
+
+##Prerequisite
+  A. EKS setup with worker nodes
+  B. Install ksonnet Quick  
+     ```
+     brew install ksonnet/tap/ks
+     ```
+  C. Get a github token
```

Comment:
Add link to https://github.com/settings/tokens?

Reply:
+1